Dockerfile

```diff
@@ -112,8 +114,8 @@ RUN ["/bin/bash", "-c", "git clone https://github.com/Kai-46/VisSatSatelliteSter
   source /opt/conda/etc/profile.d/conda.sh && \
   conda create -n vissat python=3.6 pip=20.0.* && \
   conda activate vissat && \
-  pip install -r /VisSatSatelliteStereo/requirements.txt && \
-  conda install -y -c kitware-danesfield-cf -c kitware-danesfield-df libgdal gdal"]
+  conda install -y -c kitware-danesfield-cf -c kitware-danesfield-df libgdal gdal && \
+  pip install -r /VisSatSatelliteStereo/requirements.txt"]
```

Comment:
At some point it might be worth specifying the requirements needed in `/VisSatSatelliteSterio/requirements.txt` in our `conda_env.yml`, just to keep them all in one place.  Though that's a low priority and I don't want to hold up this PR because of it.